[ADD][12.0] create stock_serial_batch_receive (#687)
OCA/stock-logistics-warehouse: stock_serial_batch_receive/readme/DESCRIPTION.rst

> This addon allows to fillup stock.move with serial number based on the given range for stock move.

Reviewer: ```suggestion
This module allows to generate automatically serial numbers on stock move lines.
```

> On v13 there will be support of serial numbers generation, for our POW, this 

Reviewer: ```suggestion
On v13 there will be support of serial numbers generation, but this 
```

> implementation is better it more robust and provide more flexibility for customization.

Reviewer: ```suggestion
implementation is better as it supports multi company and provide more flexibility for customization.
```
